app/src/test/java/com/mapzen/erasermap/view/InitActivityTest.kt

```diff
@@ -0,0 +1,72 @@
+package com.mapzen.erasermap.view
+
+import android.app.AlertDialog
```

Comment:
Holy unused imports ;)

You might want to consider enabling "Optimize Imports on the Fly" under Editor > General > Auto Import in Android Studio settings.